lib/cc/cli/prepare/quality.rb

```diff
@@ -0,0 +1,64 @@
+require "safe_yaml/load"
+
+module CC
+  module CLI
+    class Prepare
+      class Quality < Command
+        ENGINES_CONFIG = {
+          "complexity-ruby" => {
+            "enabled" => true,
+            "channel" => "beta",
+          },
+          "duplication" => {
+            "enabled" => true,
+            "channel" => "cronopio",
+            "config" => {
+              "languages" => [
+                "ruby",
```

Comment:
There are a number of answers about what to do in this `languages` list. I'm just deferring entirely for now because it's a 2 second change to do something different later.